dynamiqs/progress_bar.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import annotations
+
+
+import diffrax as dx
+from jaxtyping import PyTree, Scalar
+from jax.experimental import host_callback
+from tqdm import tqdm
```

Comment:
```suggestion
from tqdm.auto import tqdm
```
(to handle both notebooks and terminals)